sampleReact.jsx

```
@@ -0,0 +1,11 @@
+function TestComponent() {
```

ESLint

[error] 'TestComponent' is defined but never used.

(@typescript-eslint/no-unused-vars)